specs/DesignDiscussions/InputLabelsOptionLabel.md

```diff
@@ -0,0 +1,440 @@
+>
+> ```json
+> {
+>	"type": "Input.Text",
+>	"id": "textInput",
+>	"label": "Label for Input"
+> }
+> ```
+>
+> While the same rendering experience using a TextBlock would look like this
+>
+> ```json
+> {
+>	"type": "Input.Text",
+>	"id": "textInput",
+>	"label": {
+>		"type": "TextBlock",
+>		"text": "Label for Input"
+>	}
+> }
+> ```
+>
+> Both of the afore mentioned cards would produce the same rendered result that would look like  this:
+> 
+> ![img](assets/InputLabels/LabelDefault.PNG)
+>
+> It is important to note that wrapping should always be performed for this type of labels as not doing so would provide a bad user experience. If users can't read the whole label for an input they may not be able to provide a correct answer.
+
+### Host Formatting
+
+Based on the results from Q9 of the survey, we will add the option to allow the host to control formatting of the labels, either in general or for required inputs, we need a way for them to specify the desired format. There are two main things we may want to include:
+
+#### Text formatting
+ We are introducing the option to specify formatting of input labels for required and optional inputs.  We will allow full text formatting (as in the formatting properties of a TextBlock) For example:
+
+ ```json
+"inputLabelFormatting": 
+{
+	"requiredInputs": {
+		"weight": "Bolder"
+	},
+	"optionalInputs": {
+		"isSubtle":true
+	}
+}
+```
+
+The full set of properties to be modified using full text formatting will be:
+
+| Property | Default Value |
+| --- | --- | --- | --- |
+| spacing | "default" | 
+| size | "default" | 
+| color | "default" |
+
+While it's mentioned that we should allow almost all TextBlock properties to be modified by the label property, some of them must stay out of bounds for hosts as they would provide a bad user experience:
+* maxLines and wrap: Labels should be descriptive enough so users can provide the information needed, if the user can't read the whole question the information they will not be able to provide a correct answer
+* isVisible: labels should always be visible as long as the associated input is
+* height: input elements should control the height for the whole set of label, input and error message 
+
+Allowing the control of this styling via support of native styling on required and optional labels (i.e. CSS, Xaml Styles, etc) may also be required but will be set out of scope for v1 of this feature. This decision speaks to a larger discussion of the direction of host config that is beyond the scope of this document.
+
+#### Required and Optional indicators
+
+ By default, we will mark required inputs with a `*`. We may want, however, to provide the option for the host to configure a suffix to the label for required or optional inputs. This allows the host to add a `*` to required labels, or to add the words "required" or "optional". Consider a host config similar to the below:
```

Comment:
@siduppal is the default behavior we are proposing sufficient for Teams for v1? And can greater host formatting options for labels follow with v2 of inputs/label support?